operators/deploy/instance-operator/templates/clusterrole.yaml

```diff
@@ -25,6 +25,10 @@ rules:
   resources: ["deployments"]
   verbs: ["get","list","watch","create","patch","update"]
 
+- apiGroups: ["batch", "extensions"]
+  resources: ["jobs", "jobs/status"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
I would just keep the batch API since it is newer, and drop the delete verb since it is not needed given the deletion is managed through the ownerreference.